.travis.yml

```diff
@@ -21,6 +21,13 @@ after_success:
 
 jobs:
   include:
+    - stage: loopback.io
```

Comment:
When does this job run? For all builds?

Reply:
Yea, it would run for all PRs. I'm not sure how to run it if there are changes to just the `@loopback/docs` folder.

Reply:
This is pretty worrisome then.

Reply:
In the shell script, we should use `git diff --name-only` to find out if any files are changed under `docs`. If not, skip it.

Reply:
Any reason this is worrisome? Besides Travis taking longer?

I did find some inspiration for skipping a stage early here  https://github.com/facebook/react/pull/2000 I can try something similar.

Reply:
Taking longer is worrisome.

Reply:
`git diff --name-only origin/master docs/` will tell if the local repo has diff from the origin/master.

Reply:
Here is what I can see to optimize:
1. `git describe --abbrev=0` gives us the latest release tag
2. `git diff --name-only <tag> -- docs/` show docs/* changed since last release